packages/@dcl/inspector/src/components/EntityInspector/AnimatorInspector/utils.ts

```diff
@@ -21,24 +21,33 @@ export function isValidSpeed(speed: string | undefined): boolean {
   return !isNaN(parseFloat(value)) && parseFloat(value) >= 0 && parseFloat(value) <= 200
 }
 
+export function mapAnimationGroupsToStates(animations: AnimationGroup[]): PBAnimationState[] {
+  return animations.map(($) => ({
+    clip: $.name,
+    playing: !!$.isPlaying,
+    weight: $.weight ?? 1,
```

Comment:
wdyt about initializing this property as undefined and letting the user define the value in the UI?

Reply:
I remember talking with nico so about always having some defaults on the components values so user's can have their preview updated as soon as they add a component and have feedback. I think we should leave it as 1 (same value as before)